tests/all-package-defaults.nix

```diff
@@ -0,0 +1,62 @@
+{
+  lib,
+  stdenv,
+  pkgsForTest,
+  linkFarmFromDrvs,
+}:
+let
```

Comment:
Do we need a top-level comment just explaining why we have this test or what it is testing?